python/lancedb/context.py

```diff
@@ -139,6 +164,18 @@ def text_col(self, text_col: str) -> Contextualizer:
         self._text_col = text_col
         return self
 
+    def threshold(self, threshold: int) -> Contextualizer:
```

Comment:
For clarity, could we rename `threshold` to `min_window_size`? And also change the inequality to reflect that?

Reply:
Sure, definitely. That definitely makes it more clear.